pkg/v6/resource/chartconfig/resource.go

```diff
@@ -11,8 +11,8 @@ import (
 	"k8s.io/client-go/kubernetes"
 
 	"github.com/giantswarm/cluster-operator/pkg/cluster"
-	"github.com/giantswarm/cluster-operator/pkg/v6/guestcluster"
 	"github.com/giantswarm/cluster-operator/pkg/v6/key"
+	"github.com/giantswarm/guestcluster"
```

Comment:
This should be moved. There are a couple of occasions of this.

Reply:
Good catch, thanks. Fixed.